src/lib/snyk/issues.ts

```diff
@@ -21,7 +21,7 @@ const getNewIssues = (
   const debug = getDebugModule();
 
   const MonitoredIssues = snykProject;
-  debug(`Monitored snapshot had %d issues`, MonitoredIssues.length);
+  //(`Monitored snapshot had %d issues`, MonitoredIssues.length);
```

Comment:
why are you dropping this debug?

Reply:
ah that was just because I wanted less debug lines. changing that now.